cmd/crio/main.go

```diff
@@ -249,6 +249,16 @@ func main() {
 				logrus.Error(err)
 			}
 
+			// Write "$CleanShutdownFile".supported to show crio-wipe that
+			// we should be wiping if the CleanShutdownFile wasn't found.
+			// This protects us from wiping after an upgrade from a version that don't support
+			// CleanShutdownFile.
+			f, err := os.Create(config.CleanShutdownSupportedFileName())
+			if err != nil {
+				logrus.Errorf("Writing clean shutdown supported file: %v", err)
+			}
+			f.Close()
```

Comment:
if err is returned, the f will be nil

Reply:
I don't think that's a big deal. it'll return an error, but not segfault:
```
func (f *File) Close() error {
	if f == nil {
		return ErrInvalid
	}
	return f.file.close()
}

```